app/webpacker/components/MyCompetitions/UpcomingCompetitionTable.jsx

```diff
@@ -23,11 +23,18 @@ const competingStatusIcon = (competingStatus) => {
 };
 
 const registrationStatusIconText = (competition) => {
+  const toRelativeOptions = {
+    // don't be more precise than "days" (i.e. no hours/minutes/seconds)
+    unit: ["years", "quarters", "months", "weeks", "days"],
+    // round up, e.g. in 8 hours -> pads to 1 day 8 hours -> rounds to "in 1 day"
+    padding: 24 * 60 * 60 * 1000,
+  };
+
   if (competition.registration_status === 'not_yet_opened') {
-    return I18n.t('competitions.index.tooltips.registration.opens_in', { duration: DateTime.fromISO(competition.registration_open).toRelative() });
+    return I18n.t('competitions.index.tooltips.registration.opens_in', { duration: DateTime.fromISO(competition.registration_open).toRelative(toRelativeOptions) });
```

Comment:
I think in terms of Luxon respecting the locale, it's (supposed to be) as simple as passing the locale ISO code to `fromISO` like so: `DateTime.fromISO(dateIsoString, { locale: 'fr' })`. Cf. https://moment.github.io/luxon/#/intl?id=setting-the-locale

We expose the currently seleted locale as `I18n.locale` on the same I18n object that also does `t` and `tArray`.

Reply:
Interestingly, even the `toRelativeOptions` object supports a `locale` key.

Reply:
Fixed by a separate PR.